docker-compose.yml

```
@@ -1,106 +1,205 @@
+    environment:
+      - CRAWL_MODE=video
+      - START_URL=https://archive.org/details/movies
+      - MINIO_ENDPOINT=minio:9000
+      - MINIO_USER=${MINIO_ROOT_USER:-miniouser}
+      - MINIO_PASSWORD=${MINIO_ROOT_PASSWORD:-miniopassword}
+      - MINIO_BUCKET=scraped
+    depends_on:
+      - minio
+    restart: unless-stopped
+    # Single crawler with smart concurrency management
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)